clients/reality-data/src/test/integration/RealityDataClient.test.ts

```diff
@@ -106,6 +108,12 @@ describe("RealityServicesClient Normal (#integration)", () => {
   });
 
   it("should be able to retrieve the azure blob url (write access)", async () => {
+    // Skip this test if the issuing authority is not imsoidc.
+    // The iTwin Platform currently does not support the reality-data:write scope.
+    const imsUrl = await imsClient.getUrl(new ClientRequestContext(""));
+    if (-1 === imsUrl.indexOf("imsoidc"))
+      return;
```

Comment:
Can we do a `this.skip` here?  You may need to change the signature of the method to allow it from this,

```
async () => {
```

to this,

```
async function () => {
```